### DISCUSSION TITLE
After last OS update few days ago, alerts stopped working

### DISCUSSION THREAD

**Discussion by DUser26** [Security-Onion-Solutions/securityonion#13747] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

52

### RAM

512

### Storage for /

448

### Storage for /nsm

27T

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

After the last OS update and reboot, alerts stopped coming in.  all servers are shown up and running.  All was working well before the OS update. 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Do you see current data in your Elasticsearch shards? `sudo so-elasticsearch-query _cat/shards | grep suricata | sort`
